types/react-material-ui-form-validator/react-material-ui-form-validator-tests.tsx

```diff
@@ -22,12 +21,7 @@ class Test extends React.Component {
 
     render() {
         return (
-            <ValidatorForm
-                instantValidate={true}
-                debounceTime={10}
-                onSubmit={this.onSubmitted}
-                onError={this.onError}
-            >
+            <ValidatorForm instantValidate={true} debounceTime={10} onSubmit={this.onSubmitted} onError={this.onError}>
```

Comment:
Change the indentation of the params to be more readable, avoiding putting everything in one line.

Reply:
Okay, I went ahead and pushed those changes. Thanks!